### PR TITLE
FOUR-8066 Package testing with stop step

### PR DESCRIPTION
**Pull request by caleeli**
- Implement Stop step
- Include fixes

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-8066

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:deploy
ci:next
ci:processmaker:feature/FOUR-8066
ci:screen-builder:next
ci:package-testing:feature/FOUR-8066
ci:connector-send-email:next
ci:package-data-sources:next
ci:package-decision-engine:feature/FOUR-9570
ci:package-files:next
ci:package-vocabularies:feature/FOUR-8066

..

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-e91c7d0ffb.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-e91c7d0ffb.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-e91c7d0ffb.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-e91c7d0ffb.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-e91c7d0ffb.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-e91c7d0ffb.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-e91c7d0ffb.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-e91c7d0ffb.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-e91c7d0ffb.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-e91c7d0ffb.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-e91c7d0ffb.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-e91c7d0ffb.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-e91c7d0ffb.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-e91c7d0ffb.eng.processmaker.net

File: ProcessMaker/Traits/HasVersioning.php [hunk 1]
```diff
@@ -51,14 +51,16 @@ public static function saveNewVersion(Model $model)
     public function saveVersion()
```
Comment (julceslauhub) [inline ProcessMaker/Traits/HasVersioning.php:51]: add "return" section to the description

**Review by julceslauhub — APPROVED** [review]
Approved

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-e91c7d0ffb.eng.processmaker.net

**Comment by caleeli** [comment]
Hi @ryancooley, this consider this PR is ready for review and merge, is not used by anyone now.

**Pull request merged**